The v1.2.7 archive still declares version 1.2.6. Use the release version for
library, pkg-config and CMake package metadata.
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
-project(edlib VERSION 1.2.6)
+project(edlib VERSION 1.2.7)
 
 option(EDLIB_ENABLE_INSTALL "Generate the install target" ON)
 option(EDLIB_BUILD_EXAMPLES "Build examples" ON)
